css-three-column-layout/styles.css

```diff
@@ -0,0 +1,59 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  font-weight: normal;
+  color: white;
+  background-color: rgb(34, 140, 233);
+}
+
+.container {
+  width: 750px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+  text-align: center;
+}
+
+.column-third {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+img {
+  width: 30%;
+  background-color: white;
+
+  /* Googled and found the clip-path property */
+  clip-path: circle(50%);
```

Comment:
This totally works! You could have also put the image in a `div` that you made into a circular white element, then add the image as a child of that `div`

Reply:
Well noted. Thank you!